### ISSUE TITLE
Data migration script to copy data to updated contracts

### ISSUE THREAD

**Issue by bumi** [67P/kredits-contracts-legacy#13]
As contracts are immutable but we want to iterate developing our contracts we need a method to migrate data from the "old" contract to a newly deployed contract. This will be useful in the current experimental stage.
A simple and quick solution could be a script like the `seed.js` script that reads data from the old contract and populates the new contract. 

In the future we need to get rid of such a centralized, off-chain script but for now this would be a quick solution.

**Comment by bumi** [comment]
I think the embark-framework has some kind of "migrations". I am not sure what it exactly does but the name suggest that it might be worth looking into it.

**Comment by raucao** [comment]
Wouldn't it be easier to just implement the splitting out of the data to super simple contracts, that we don't have to change anymore? Our actual data now mostly lives in IPFS anyway, so for our current problem of losing e.g. contributors, I think that specific contract could be simple enough, so we don't have to migrate data after publishing it. But maybe I'm missing sth.

**Comment by bumi** [comment]
yeah that would be the overall goal. 
But I still would think it would be a good idea to somehow have the chance to for example get the contributor IPFS hashes into a possible new contract.

**Comment by raucao** [comment]
Yes, probably. I'm just trying to figure out if it makes sense to spend the time on that script instead of on splitting out the data, so it's better from the start, with the same effort, and we don't have planned manual migrations by design.

**Comment by bumi** [comment]
I had experimented with this [seed script](https://github.com/67P/kredits-contracts/blob/master/scripts/seed.js) that basically consumes a JSON and calls methods on the contract. 
If the contract supports that (for example: that the owner is allowed to set certain data) not much would be missing to copy some data. 
but I agree that this is not the goal and not what we want. - more hack until we're there.
